src/AppBundle/Resources/views/Default/rulesreference.en.html.twig

```diff
@@ -667,7 +667,7 @@ span.new	{ color:red }
 <ul>
 <li>  Sometimes a keyword is followed by reminder text, which is presented in italics. Reminder text is a shorthand explanation of how a keyword works, but it is not rules text and does not replace the rules for that keyword in this glossary.</li>
 <li>  Cards are considered to have a keyword or to not have that keyword. A single card that has and/or is gaining the same keyword from multiple sources functions as if it has one instance of that keyword. (Note: Instances of Ambush (X) with different values for X are considered different keywords.)</li>
-<li>  The keywords in the game are: ambush (X), insight, intimidate, limited, no attachments, pillage, renown, stealth, and terminal.</li>
+<li>  The keywords in the game are: ambush (X), insight, intimidate, limited, no attachments, pillage, renown, shadow (X), stealth, and terminal.</li>
```

Comment:
It might be worth adding bestow (X) to this list - apparently was missed when the tooltip for Bestow was added.

Reply:
handled.